### PR TITLE
testing/wine-mono: new port

### PR DESCRIPTION
**Pull request by staalmannen**
http://wiki.winehq.org/Mono
Open source .NET replacement

### CODE REVIEW

File: testing/wine-mono/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+license="GPL"
+depends="wine"
+makedepends=""
+install=""
+subpackages=""
+source="https://dl.winehq.org/wine/wine-mono/${pkgver}/wine-mono-${pkgver}.msi"
+
+_builddir=
+prepare() {
+	mkdir -p "$srcdir"
```
Comment (jirutka) [inline testing/wine-mono/APKBUILD]: `$srcdir` always exists, what’s the point of this line?

File: testing/wine-mono/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+pkgrel=0
+pkgdesc="Wine's built-in replacement for Microsoft's .NET Framew"
+url="http://wiki.winehq.org/Mono"
+arch="any"
+license="GPL"
+depends="wine"
+makedepends=""
+install=""
+subpackages=""
+source="https://dl.winehq.org/wine/wine-mono/${pkgver}/wine-mono-${pkgver}.msi"
```
Comment (jirutka) [inline testing/wine-mono/APKBUILD]: How is this binary built? Is it really platform-independent?

File: testing/wine-mono/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Maintainer: Jens Staal <staal1978@gmail.com>
+pkgname=wine-mono
+pkgver=4.6.3
+pkgrel=0
+pkgdesc="Wine's built-in replacement for Microsoft's .NET Framew"
+url="http://wiki.winehq.org/Mono"
+arch="any"
```
Comment (jirutka) [inline testing/wine-mono/APKBUILD]: This is not a valid type. If this binary is platform-independent, then it should be `noarch`.

**Comment by jirutka** [comment]
@ncopa @clandmeter Is it okay to package binaries like this one?

File: testing/wine-mono/APKBUILD (hunk 3 above)
Comment (staalmannen) [inline testing/wine-mono/APKBUILD]: OK thanks I will change.

File: testing/wine-mono/APKBUILD (hunk 1 above)
Comment (staalmannen) [inline testing/wine-mono/APKBUILD]: no point at all. I copied the APKBUILD from wine_gecko and this must have stayed there... I will remove it.

**Comment by ncopa** [comment]
yes, its accptable since its a windows binary to run within wine. We don't dont have any way to build windows binaries on alpine (and I see no reason to why we should implement it).

**Comment by jirutka** [comment]
I'm quite sceptic about usefulness of such package. Also is it legally okay?

**Comment by ncopa** [comment]
fixed with 32b83bc18acf619bd9f4695370f35b3f6c9af6e3

**Pull request closed without merging**